rest/admin_api.go

```diff
@@ -189,11 +189,10 @@ func (h *handler) handleGetDbConfig() error {
 	if !includeJavascript {
 		responseConfig.Sync = nil
 		responseConfig.ImportFilter = nil
-		for _, evt := range responseConfig.EventHandlers.DBStateChanged {
-			evt.Filter = ""
-		}
-		for _, evt := range responseConfig.EventHandlers.DBStateChanged {
-			evt.Filter = ""
+		if responseConfig.EventHandlers != nil && responseConfig.EventHandlers.DBStateChanged != nil {
+			for _, evt := range responseConfig.EventHandlers.DBStateChanged {
+				evt.Filter = ""
+			}
```

Comment:
You can range over a nil slice and it will just do zero iterations.
https://play.golang.org/p/YYiqD8cup_a

Turns out the original code was supposed to be iterating over both types of event handler, so just having the `EventHandlers` nil check should be enough:

```suggestion
		if responseConfig.EventHandlers != nil {
			for _, evt := range responseConfig.EventHandlers.DocumentChanged {
				evt.Filter = ""
			}
			for _, evt := range responseConfig.EventHandlers.DBStateChanged {
				evt.Filter = ""
			}
```